spec/locales-currencies-tz.html

```diff
@@ -164,59 +164,229 @@ <h1>
+      </p>
+    </emu-note>
 
     <p>
-      A conforming implementation must recognize *"UTC"* and all other Zone and Link names (and <strong>only</strong> such names), and use best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations. However, the set of combinations of time zone name and language tag for which localized time zone names are available is implementation dependent.
+      The IANA Time Zone Database is typically updated between five and ten times per year.
+      These updates may add new Zone or Link names, may change Zones to Links, and may change the UTC offsets and transitions associated with any Zone.
+      ECMAScript implementations are recommended to include updates to the IANA Time Zone Database as soon as possible.
+      Such prompt action ensures that ECMAScript programs can accurately perform time-zone-sensitive calculations and can use newly-added available named time zone identifiers supplied by external input or the host environment.
+    </p>
+    <p>
+      Although the IANA Time Zone Database maintainers strive for stability, in rare cases (averaging less than once per year) a Zone may be replaced by a new Zone.
+      For example, in 2022 "*Europe/Kiev*" was deprecated to a Link resolving to a new "*Europe/Kyiv*" Zone.
+      The deprecated Link is called a <dfn variants="renamed time zone identifiers">renamed time zone identifier</dfn> and the newly-added Zone is called a <dfn variants="replacement time zone identifiers">replacement time zone identifier</dfn>.
+    </p>
+    <p>
+      To reduce disruption from these infrequent changes, ECMAScript implementations must initially add each replacement time zone identifier as a non-primary time zone identifier that resolves to the existing renamed time zone identifier.
+      This allows ECMAScript programs to recognize both identifiers, but also reduces the chance that an ECMAScript program will send the replacement time zone identifier to another system that does not yet recognize it.
+      After a waiting period, implementations must promote the new Zone to a primary time zone identifier while simultaneously demoting the renamed time zone identifier to non-primary.
+      This waiting period is two years after the IANA Time Zone Database release containing the changes, to provide ample time for other systems to be updated.
+      After two years, implementations should update their time zone data to make the replacement time zone identifier primary and the renamed time zone identifier non-primary.
+      This two-year period does not need to be exact.
+      For example, it's acceptable to wait until the next ICU release after two years has expired.
```

Comment:
Cldr release?

Reply:
Are they released separately?  I thought (but if I'm mistaken let me know!) that each ICU release bundled a specific version of CLDR, so from the point of view of an ECMAScript engine, it's the ICU release that they will include in their builds and CLDR data comes along with that. But I'm admittedly not familiar with the release process so I might have gotten it wrong.

Would "ICU and CLDR release" be better here?

Reply:
@srl295 Do you know answers to the questions above? Thanks!

Reply:
There are three ICU projects (ICU4C, ICU4J, and ICU4X). CLDR and ICU4C/ICU4J releases are sync'ed, but ICU4X has a different release cycle. ECMAScript implementations use all three ICU projects. For example ICU4C is used by JSC, V8, and SpiderMonkey; ICU4X is used by SpiderMonkey for `Intl.Segmenter`; ICU4J is used by GraalJS. 

So to avoid confusion which ICU project is meant, referring to CLDR is preferable.

Reply:
Not only that, the spec explicitly avoids referring to specific ICU/non-ICU implementations in favor of referring to CLDR.

Reply:
OK, I'll replace ICU with CLDR here, and elsewhere.